src/pages/operators/index.tsx

```diff
@@ -693,7 +699,7 @@ const styles = (theme: Theme) => css`
       ${theme.breakpoints.up("mobile")} {
         display: grid;
         grid-auto-flow: column;
-        grid-template-columns: repeat(3, auto) 1fr;
+        grid-template-columns: repeat(4, auto) 1fr;
```

Comment:
Does this layout still look correct when the Reset Filters button isn't rendered (since there'll be one fewer child element)? I can take a look rq...

Reply:
Oh yep, it's fine since `label.guide-available-checkbox` has `grid-column: -1`. Never mind then!